src/xia2/Modules/MultiCrystal/data_manager.py

```diff
@@ -205,6 +205,18 @@ def export_unmerged_wave_mtz(self, wl, prefix, d_min, wavelength_tolerance):
             return params.mtz.hklout
         return None
 
+    def export_unmerged_wave_mmcif(self, wl, prefix):
+        data = self.data_split_by_wl[wl]
+        nn = len(self.wavelengths)
+        fmt = "%%0%dd" % (math.log10(nn) + 1)
+        index = sorted(self.wavelengths.keys()).index(wl)
+        params = export.phil_scope.extract()
```

Comment:
I think here we should also set the d_min value to make sure we are exporting the same resolution range. The dials export_mmcif code actually uses the `mtz.d_min` parameter to filter the data (yes that is unexpected and a bad design).